crates/biome_service/src/workspace/server.rs

```diff
@@ -783,10 +801,22 @@ impl Workspace for WorkspaceServer {
     }
 
     fn search_pattern(&self, params: SearchPatternParams) -> Result<SearchResults, WorkspaceError> {
+        let SearchPatternParams { path, .. } = params;
+
         // FIXME: Let's implement some real matching here...
```

Comment:
Is the `FIXME` still applicable? I see there's some new code, maybe the `FIXME` should become a TODO or something else?

Reply:
A `TODO` definitely sounds better, thanks! 👍